opengever/document/tests/test_indexers.py

```diff
@@ -550,3 +550,32 @@ def test_related_items_is_updated_when_related_item_is_deleted(self, browser):
         self.assertTrue(relations[0].isBroken())
         # Index should only contain unbroken relations
         self.assertEqual(None, solr_data_for(self.document, 'related_items'))
+
+    def test_document_version_count_indexed(self):
+        self.login(self.regular_user)
+        versioner = Versioner(self.document)
+        versioner.create_version('Initial version')
+        versioner.create_version('Second version')
+        self.commit_solr()
+
+        indexed_value = solr_data_for(self.document, 'document_version_count')
+        self.assertEqual(indexed_value, self.document.get_current_version_id())
```

Comment:
Still keep the assertion for the indexed_value to see explicitly which version is indexed as you did for mails:

```suggestion
        self.assertEqual(indexed_value, 1)
        self.assertEqual(indexed_value, self.document.get_current_version_id())
```

Reply:
✔️